sonic_package_manager/manager.py

```diff
@@ -1002,7 +1003,7 @@ def get_manager() -> 'PackageManager':
         docker_api = DockerApi(docker.from_env(), ProgressManager())
         registry_resolver = RegistryResolver()
         metadata_resolver = MetadataResolver(docker_api, registry_resolver)
-        cfg_mgmt = config_mgmt.ConfigMgmt(source=INIT_CFG_JSON)
```

Comment:
I did not understand this use. 
So ConfigMgmt searches for yang models at a fixed location 'YANG_DIR = "/usr/local/yang-models"', then in what case it is an error ?.

Kindly elaborate, thx.

Reply:
Not exactly 
ConfigMgmt searches for yang models in current working directory AND "/usr/local/yang-models"

Reply:
Since the expectation is to only read schema from `/user/local/yang-models` let's disable reading schema from CWD (current working dir) in sonic-mgmt directly, no need to create a flags object and pass it.

libyang ref: https://netopeer.liberouter.org/doc/libyang/master/html/group__contextoptions.html
PR to add SonicYangOptions needs to be reverted: https://github.com/Azure/sonic-buildimage/pull/10359

Reply:
@ghooo  I have considered this option earlier, but after the discussion I chose the current approach. It looks more flexible to me and allows to pass different yang options to sonic-yang-mgmt